nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -993,6 +954,229 @@ Expressions will provide the following results:
 
 
 
+[[escape]]
```

Comment:
This section might better be called "Encode/Decode", since escaping is a form of encoding and the urlEncode/Decode functions have been moved here.